### ISSUE TITLE
Runtime exception when using multiple WithBearerAuthentication

### ISSUE THREAD

**Issue by softexpert** [Jaxelr/Nancy.Metadata.OpenApi#66]
Hello,
# Describe the bug

In my MetadataModule implementation I have declared  several OpenApiRouteMetadata that, each, have a WithBearerAuthentication description. At runtime I get the error
> System.ArgumentException: 'Can not add property bearer to Newtonsoft.Json.Linq.JObject. Property with the same name already exists on object.'

This occur in the CustomCollectionJsonConverter.cs file at line 26.

## Steps to reproduce

Using the following example MetadataModule allows to highlight the issue

```
public class RootMetadataModule : MetadataModule<OpenApiRouteMetadata>
{
	public RootMetadataModule()
	{

		Describe["ListAvailableProcesses"] = desc => new OpenApiRouteMetadata(desc)
			.With(i => i.WithResponseModel("200", typeof(ProcessesContainer), "List available processes")
						.WithSummary("a description"));

		Describe["SendProcessingRequest"] = desc => new OpenApiRouteMetadata(desc)
			.With(info => info.WithResponseModel("200", typeof(ProcessingResponse), "Request was accepted ")
				.WithResponse("400", "Bad request")
				.WithSummary("POST example")
				.WithRequestModel(typeof(ProcessingRequest), contentType: "application/json", description: "Send processing request")
				.WithBearerAuthentication("JWT", "Please enter the following security token «bla bla»")
				);

		Describe["QueryProcessingState"] = desc => new OpenApiRouteMetadata(desc)
			.With(info => info.WithResponseModel("200", typeof(ProcessingStateResponse), "Request was accepted and processing state is sent back")
				.WithResponse("400", "Bad request")
				.WithSummary("POST example with ProcessingStateRequest model")
				.WithRequestModel(typeof(ProcessingStateRequest), contentType: "application/json", description: "Send processing state request", required: true)
				.WithBearerAuthentication("JWT")
				);

	}
}
```

## Expected behavior

No exception should occur

# Proposed Fix
At line 26 in CustomCollectionJsonConverter.cs, if I add the following code, the duplicate key exception is averted; not sure, however, if this does not break something else ...
```
if (jo.ContainsKey(t.Key))
{
	continue;
}
```

- Dotnet version: [dotnet472]
- Library version: [current GIT version]

**Comment by Jaxelr** [comment]
Thanks for reporting, ill take a look in the following days. 👍

**Comment by Jaxelr** [comment]
This is fixed on the develop branch. I will proceed to bump version, tag and create myget package, then do some testing before publishing on nuget by tomorrow. 

Thanks!

**Comment by Jaxelr** [comment]
Just published on nuget version 0.8.2 that corrects this bug. Feel free to reopen Issue if you see anything from your end.
